content/09.evolution.md

```diff
@@ -69,31 +69,98 @@ Research in response to prior outbreaks of HCoV-borne infections, such as SARS a
+Because of the challenges associated with analyzing such a variable region, it was excluded from the large-scale COVID-19 HGI GWAS analysis [@doi:10.1038/s41586-021-03767-x].
+
+Finally, significant attention has been paid to the question of whether ABO blood type is associated with COVID-19 outcomes.<!--This paper is interesting but I'm not totally sure how to incorporate it 10.1371/journal.pone.0251535-->
+ABO blood type has been found to modulate susceptibility to other pathogens [@doi:10.1001/jama.2020.16516].
+While ABO blood type is a genetic trait, it is more easily evaluated than the genetic regions discussed above because of the simple relationship between genetic variants and phenotype.
+The possibility for an association between blood type and COVID-19 infection was raised early in the pandemic in a preprint that reported associations in 2,173 patients in Wuhan and Shenzhen, China [@doi:10.1101/2020.03.11.20031096].
+The protective effect of O and increased risk associated with A blood types that they reported was subsequently investigated by many studies that returned varied results (e.g., [@doi:10.1002/jmv.27444; @doi:10.3389/fcimb.2020.00404; @doi:10.1182/bloodadvances.2020002623; @doi:10.1001/jamanetworkopen.2021.7429]; see [@doi:10.1007/s00277-021-04489-w] for a literature review).
+Support for higher and lower risk, respectively, of SARS-CoV-2 infection with A and O blood types was supported by a meta-analysis [@doi:10.1016/j.meegid.2020.104485].
+While the support for the association was independent of a mechanism, a possible relationship between ACE activity and blood type has been proposed [@doi:10.1177/2047487320922370] as has an effect on carbohydrate-carbohydrate interactions relevant to ACE2 binding [@doi:10.1016/j.mehy.2020.110155].
+This is the only candidate gene described that has received additional support from GWAS, as is discussed below.
+
+The COVID-19 literature related to candidate gene investigations demonstrates relatively low inter-study consistency in findings.
+In particular, sample size is a major challenge in designing these studies.
+However, for many traits, the relationships between genes and phenotypes are complex, and selecting which variants to sequence is not always straightforward
+.
+As a result, in the age of next-generation sequencing, discovery-driven studies have emerged as an alternative approach.
+
+##### Genome-Wide Association Studies
+
+Genome-wide association studies (GWAS) offers a discovery-driven approach that provides a different perspective than candidate gene studies.
+Instead of selecting a gene or variant _a priori_, in GWAS, a large number SNPs (usually several million) are evaluated at once to identify those most likely to vary in correlation with a trait of interest.
```

Comment:
```suggestion
Instead of selecting a gene or variant _a priori_, in GWAS, a large number of SNPs (usually several million) are evaluated at once to identify those most likely to vary in correlation with a trait of interest.
```